src/content/blog/windows-ddev-setup.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Setting up a Windows Machine for DDEV"
+pubDate: 2024-11-04
+#modifiedDate: 2024-10-17
+summary: Setting up a new Windows machine for DDEV maintenance or development is pretty easy. Here are my opinionated steps.
+author: Randy Fay
+featureImage:
+  src: /img/blog/2024/11/windows-install-blog-logos.png
+  alt: Windows, DDEV, Ubuntu logos demonstrating setting up a Windows machine for DDEV.
+categories:
+  - DevOps
+---
+
+I've recently set up a few Windows machines for DDEV maintenance and development, and wanted to share how I do it. It's surprisingly easy. My approach here is opinionated, but it works for me. You'll do things a little differently I'm sure.
+
+Two recent Windows machines I set up were the new ARM64/Qualcomm/CoPilot variety. They were excellent and fast and had great battery life. There's very little I had to do differently with them, but I'll mention in the steps when there was something different. (I was surprised by the lack of a fingerprint sensor on both, but the Windows Hello facial recognition was quite fast. There is no ARM64 Discord app, and the AMD64 one had horrific performance.)
+
+1. Remove unwanted applications. I usually start by removing a bunch of bloatware. 
+2. Do all Windows updates. 
+3. Turn off the `System` -> `Notifications` -> `Additional settings` that cause the “Windows experience” prompts after upgrade. 
+4. Enable Windows Update->Advanced Options->Receive updates for other Microsoft Products. Amazingly, this is not on by default, and you might have an old WSL2 kernel! (See [Beware of Dirty Pipes](beware-of-dirty-pipes-and-docker-desktop-on-windows.md).)
+5. Install important apps. I always start with these. Each of these except Discord had an ARM64 version. If on an ARM64 machine, make sure you get the right version!
+    -  Notion
+    -  Chrome
+    -  1Password
+    -  PhpStorm (Although there are many ways to use PhpStorm on WSL2, I just open the project in `\\wsl.localhost\Ubuntu\home\rfay\workspace\<project>` and it works great and performance is fine.)
+    -  GoLand
+    -  Discord
+    -  Slack
+    - [Perforce P4V](https://www.perforce.com/downloads/helix-visual-client-p4v) is my favorite merge conflict resolver.
+6. In PowerShell, `wsl --install` and `wsl --update`
+7. Windows Terminal is a fantastic terminal and is installed by default these days. I always set it up early with "Default Terminal Application: Windows Terminal" and "Interaction->Automatically Copy Selection to Clipboard", and set Ubuntu as default, and have it auto-start on login. 
```

Comment:
not a windows user so i am unfamiliar with the details but the detail that puzzles me is, point 6 is about powershell and then point 7 is all of a sudden about windows terminal which is, i suppose, something different?

Reply:
PowerShell is a shell, Windows Terminal is a way to access it. Like iTerm2 is a terminal, bash/zsh are shells.